### PR TITLE
Avatar badge branch

### PR DESCRIPTION
**Pull request by zoef-yt**

### CODE REVIEW

**Comment by tonymike99** [comment]
Looks great!

**Review by dcs-soni — COMMENTED** [review]
Maybe you can try to arrange the CSS properties inside the selectors in the lexicographic order to maintain consistency and readability.  It would help if the code base of CSS becomes too big.

And everything looks great 👍

**Comment by zoef-yt** [comment]
> Maybe you can try to arrange the CSS properties inside the selectors in the lexicographic order to maintain consistency and readability. It would help if the code base of CSS becomes too big.
> 
> And everything looks great 👍

Okay, thanks will make those changes. Should I merge after those changes?

**Review by sanu29 — APPROVED** [review]
All good to proceed with commit according to me. Great work could not find any issues.
Spacing is great as well your naming conventions are meaningful and easy.
File structure is also good. Min styles applied and min lines of code kept in each CSS file.
Great work.

File: pages/components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.avatar {
+	border-radius: 50%;
+	object-fit: cover;
+	margin: 1rem;
+	width: 5rem;
+	height: 5rem;
+}
+.avatar-xlg {
```
Comment (varun13169) [inline pages/components/avatar/avatar.css:9]: Nit: spacing

Line 7-8 vs line 11-13

File: pages/components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	padding: 2px;
+	top: 8px;
+	right: 5px;
+	height: 1.5rem;
+	width: 1.5rem;
+	background: var(--colour-error);
+	color: var(--colour-text-white);
+	border-radius: 50%;
+}
+.badge-status {
```
Comment (varun13169) [inline pages/components/badge/badge.css:21]: Nit: spacing

File: pages/components/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	border: 2px solid var(--colour-black-one);
+}
+
+.badge-status.online {
+	background: var(--colour-accent-one);
+}
+.badge-status.dnd {
+	background: var(--colour-error);
+}
+.badge-status.idle {
```
Comment (varun13169) [inline pages/components/badge/badge.css:44]: Nit: spacing

**Review by varun13169 — CHANGES_REQUESTED** [review]

**Pull request merged**